env-example

```diff
@@ -0,0 +1,8 @@
+GITHUB_TOKEN1=
+GITHUB_TOKEN2=
+GITHUB_API=https://api.github.com/graphql
+ENABLE_REDIS=
+REDIS_PORT=10709
```

Comment:
Why is port 10709?

Reply:
I think the port is better to match with the docker compose port.

Reply:
Good catch, changed.